sessions/Sources/App/Models/User.swift

```diff
@@ -58,16 +71,17 @@ struct CreateUserRequest: Decodable {
 
 /// User encoded into HTTP response
 struct UserResponse: HBResponseCodable {
-    let id: UUID?
+    let id: UUID
     let name: String
 
-    internal init(id: UUID?, name: String) {
+    internal init(id: UUID, name: String) {
         self.id = id
         self.name = name
     }
 
-    internal init(from user: User) {
-        self.id = user.id
+    internal init(from user: User) throws {
+        guard let id = try? user.requireID() else { throw HBHTTPError(.unauthorized) }
```

Comment:
I propose re-throwing the error from Fluent's `requireID()` here

Reply:
see above